### PR TITLE
Phone page adaptive layout (without font)

### PR DESCRIPTION
**Pull request by ihorhladkov**

### CODE REVIEW

File: src/Components/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/* eslint-disable max-len */
+import React, { useState } from 'react';
+import arrowDown from '../assets/Icons/Chevron (Arrow Down).svg';
+import arrowUp from '../assets/Icons/Chevron (Arrow Up).svg';
```
Comment (olena-ole) [inline src/Components/Dropdown.tsx:4]: use react-icons lib ([Feather set](https://react-icons.github.io/react-icons/icons?name=fi) )

**Review by olena-ole — DISMISSED** [review]
GJ! Just use icons and make sure your styles correspond to the updated tailwind config

File: src/pages/PhonePage.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    value: '16',
+    label: '16',
+  },
+];
+
+const PhonePage: React.FC = () => {
+  return (
+    <main className="container mx-auto flex flex-col items-center tablet:items-start px-4 pt-6 tablet:px-6 desktop:w-[1200px]">
+      <div>
+        <div className="mb-8">
```
Comment (MaxSchmide) [inline src/pages/PhonePage.tsx:37]: Check this divs please

**Review by MaxSchmide — CHANGES_REQUESTED** [review]

**Review by MaxSchmide — DISMISSED** [review]

**Review by NazarBaraban — DISMISSED** [review]

**Comment by MaxSchmide** [comment]
Please update with master branch

**Review by olena-ole — APPROVED** [review]

**Review by NazarBaraban — APPROVED** [review]

**Pull request merged**